ctru-rs/examples/file-explorer.rs

```diff
@@ -137,7 +143,7 @@ impl<'a> FileExplorer<'a> {
         }
     }
 
-    /// Paginate output
+    /// Paginate output.'
```

Comment:
IIRC doc comments on non-`pub` functions are the same as regular comments, and I think it actually might emit a clippy warning in some cases to have three slashes like this

```suggestion
    // Paginate output.
```